quic/s2n-quic-core/src/recovery/sent_packets.rs

```diff
@@ -66,20 +75,24 @@ mod test {
         frame::ack_elicitation::AckElicitation,
         inet::ExplicitCongestionNotification,
         path,
-        recovery::SentPacketInfo,
+        recovery::{bandwidth, SentPacketInfo},
         time::{Clock, NoopClock},
     };
 
     #[test]
     #[should_panic]
     fn too_large_packet() {
+        let bw_estimator = bandwidth::Estimator::new(NoopClock.get_time());
```

Comment:
NoopClock.. seems test related and wrong here.

Reply:
this is a test, though.